README.md

```diff
@@ -65,7 +65,7 @@ the `data` directory; you can access it at `/data/<filename>.json`.
 
 ### Available dependencies
 
-The repo includes jQuery, Backbone, Underscore, and RequireJS. If there's other
+The repo includes jQuery, Backbone, and Underscore. If there's other
 stuff you'd find useful, you can put it in the `lib` directory.
```

Comment:
maybe tell people to use `npm` to add things if they need? they would also have to add the dep to `app/runner.html` as well as whatever answers they would use it in using node's `require`.